client/src/components/FileBrowser.vue

```diff
@@ -53,6 +56,17 @@
               mdi-download
             </v-icon>
           </v-btn>
+          <v-btn
+            icon
+            @click="setFile(item)"
```

Comment:
Should this be `setOpenFile`? I can't fine `setFile` anywhere else

```suggestion
            @click="setOpenFile(item)"
```